lib/packages/core/lib/src/views/media/media_forwarding.dart

```diff
@@ -56,7 +55,6 @@ class _LMChatMediaForwardingScreenState
 
   @override
```

Comment:
we can remove this override, since ` LMChatMediaHandler.instance.clearPickedMedia();` have been removed from here.

Reply:
Oh yes right, done